62.md

```diff
@@ -38,6 +38,8 @@ Relays MAY store the signed deletion request for bookkeeping.
 
 Paid relays or relays that restrict who can post MUST also follow the request regardless of the user's status. 
 
+Kind `5` deletions MUST not delete kind `62`s. 
```

Comment:
Should we add that kind `62` should not delete kind `62` for the same reasons?

Reply:
I think it's fine that a newer 62 deletes an older 62. It effectively supercedes it

Reply:
So is it implied that kind `62` events should be kept and be the only event that is kept in the relay for the deleted pubkeys? If that's the case it seems there's no clarification about that in the copy. In other words, `MUST not delete kind 62s` seems to hint that the vanish requests should be kept. Maybe the intent was that it should be kept until being processed?

Reply:
The idea is that the vanishing request cannot be undone. Whether the event itself is kept by the relay should be an implementation detail.
@vitorpamplona perhaps this could be reworded same way as in NIP-09 where it says `Publishing a deletion request event against a deletion request has no effect. Clients and relays are not obliged to support "unrequest deletion" functionality.`?

Reply:
Just adjusted